Flutter/calendar/drag-drop.md

```diff
@@ -49,7 +49,7 @@ Widget build(BuildContext context) {
 [resource](https://pub.dev/documentation/syncfusion_flutter_calendar/latest/calendar/AppointmentDragStartDetails/resource.html) - Get the resource details.
 
 {% tabs %}
-{% highlight Dart %}
+{% highlight dart hl_lines="9 10" %}
```

Comment:
Check 9 is needed or not and highlight method also

Reply:
method highlighted.